ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,7 @@
+<!--- STOP! Before you open an issue please search this repositories issues to see if has already been reported. This helps reduce duplicate issues being from being. -->

```

Comment:
repositories -> repository's
to see if has already been reported -> to see if it has already been reported
This helps reduce duplicate issues being from being -> This helps reduce duplicate issues from being created
lmao a few too many 🍻 ?

Reply:
I'll make those changes right now. I was trying to do too many things at once here in the office.